requirements/quality.txt

```diff
@@ -1,3 +1,9 @@
-pep8==1.7.0
+astroid==1.6.5; python_version < "3.0"
+astroid==2.1.0; python_version >= "3.0"
+pycodestyle==2.5.0
+pylint==1.9.4; python_version < "3.0"
+pylint==2.2.2; python_version >= "3.0"
```

Comment:
I'm surprised this works given that edx_lint 0.5.5 depends on specific versions of astroid, pylint, and pylint-django, but Travis does seem to be installing the correct versions.